tests/unit/pywbemcli/simple_foo_mock_model.mof

```diff
@@ -0,0 +1,224 @@
+// This is a simple mof model that creates the qualifier declarations,
+// classes, and instances for a very simplistic model to be used in the
+// pywbemcli mock test environment.
```

Comment:
Should probably mention that that this is based on multiple namespaces, interop and user namespace.

Reply:
The MOF file does not know this. I added that this mof model is used for the user namespace.